argoverse/data_loading/argoverse_tracking_loader.py

```diff
@@ -25,7 +26,7 @@ def __init__(self, root_dir: str) -> None:
         self.CAMERA_LIST = CAMERA_LIST
         self._log_list: Optional[List[str]] = None
         self._image_list: Optional[Dict[str, Dict[str, List[str]]]] = None
-        self._image_list_sync: Optional[Dict[str, Dict[str, List[np.ndarray]]]] = None
+        self._image_list_sync: Optional[Dict[str, Dict[str, List[NDArray[np.float64]]]]] = None
```

Comment:
Hi Anna, the image timestamps should be in nanoseconds (integers), so likely we need to cast to np.int32 or np.int64 somewhere beforehand